usefulUtilities/hiFiCalendar/clientScripts/meetingRoomSetup_app.js

```diff
@@ -0,0 +1,161 @@
+                });
+                break;          
+            case "GET AVAILABLE ROOMS":
+                ui.sendToHtml({
+                    type: "AVAILABLE ROOMS",
+                    roomConfig: roomConfigInfo
+                });
+                break;
+            case "SETUP COMPLETE":
+                token = data.access_token;
+                refreshToken = data.refresh_token;
+                tokenLifetime = data.expireTime;
+                tokenStart = data.validSince;
+                expireTime = tokenStart + tokenLifetime * MS_TO_SEC;
+                timezone = data.timeZoneName;
+                // Gets the correct timezone offset for events current / in the future, which is why the -1 is needed.
+                timezoneOffset = (new Date().getTimezoneOffset() / MIN_PER_HR) * -1; 
+                roomConfig = data.connectionData;
+                Settings.setValue("roomConfigured", true);
+                Entities.callEntityServerMethod(TOKEN_SERVER_ID, "initializeRooms", [
+                    token, 
+                    refreshToken, 
+                    expireTime, 
+                    timezone,
+                    timezoneOffset, 
+                    JSON.stringify(roomConfig)
+                ]);
+                break;
+        }
+    }
+
+
+    // This function is the listener for messages on the message mixer related to this app.
+    var messageHandler = function(channel, message, senderUUID, localOnly) {
+        if (channel !== CHANNEL) {
+            return;
+        } else {
+            try {
+                message = JSON.parse(message);
+            } catch (e) {
+                console.log(e, "Could not parse message");
+                return;
+            }
+
+            if (message.TOKEN_SERVER_ID) {
+                TOKEN_SERVER_ID = message.TOKEN_SERVER_ID;
+            }
+
+            if (message.type === "TOKEN EXPIRED") {
+                Window.alert("The authorization token for your Google Calendar could not be refreshed.\n" + 
+                "Please open your calendar app and reauthorize to continue displaying calendar schedules.");
+                roomConfig = message.roomConfig;
+                roomConfigInfo = message.roomConfigInfo;
+            } else if (message.type === "STATUS UPDATE") {
+                if (message.tokenStatus) {
+                    Window.announcement("Access tokens are valid. No action required");
+                    roomConfig = message.roomConfig;
+                    roomConfigInfo = message.roomConfigInfo;
+                } else if (!message.tokenStatus && !message.roomConfigured) {
+                    roomConfigInfo = message.roomConfigInfo;
+                    Window.announcement("You have not set up any room schedules for this domain.");
+                }
+            } else if (message.type === "ERROR") {
+                // Commenting this out for right now because this was initially flooding the interface during testing
```

Comment:
should this be brought back?

Reply:
I think so, a little nervous about it originally because when I first was doing this, they were coming out as the prompts.  I noticed though that all of these started becoming the transparent messages on the side so they weren't too bad.  I'm going to bring this one back but will ask Salina to keep an eye out for it in smoke tests to see if this is more annoying than helpful.

Reply:
Does it need to be a `Window.alert` or can it be a console log?

Reply:
I changed it to an announcement.  Was a little confused just now and thought they were all announcements until i reviewed back and saw it was an alert.  I think at least that way if someone sees it, they can just report it easier and it's not in the way like the other messages.

Reply:
I actually like all of these as logs instead per the next comment.